autometrics/src/labels.rs

```diff
@@ -246,3 +228,75 @@ pub trait GetStaticStr {
     }
 }
 impl_trait_for_types!(GetStaticStr);
+
+#[macro_export]
+macro_rules! result_labels {
```

Comment:
The reason it's not in the macros crate is mostly that it's just expanding code and that I wanted the macro to access values from `autometrics` (the constants, the labels, the traits): since the macro is not a proc macro returning an arbitrary TokenStream, we need to have proper access to the crate (which we don't have inside `autometrics-macros`)